.travis.yml

```diff
@@ -6,31 +6,31 @@ script:
 matrix:
   fast_finish: true
   include:
-  - rvm: 2.1.6
+  - rvm: 2.3.1
     dist: trusty
     env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true
     script: bundle exec rake beaker
     services: docker
     sudo: required
-  - rvm: 2.1.6
+  - rvm: 2.3.1
     dist: trusty
     env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=ubuntu-16.04
     script: bundle exec rake beaker
     services: docker
     sudo: required
-  - rvm: 2.1.6
+  - rvm: 2.3.1
     dist: trusty
     env: PUPPET_INSTALL_TYPE=agent BEAKER_debug=true BEAKER_set=centos-7
     script: bundle exec rake beaker
     services: docker
     sudo: required
-  - rvm: 2.1.6
+  - rvm: 2.3.1
```

Comment:
The puppet 4 all in one package bundles Ruby 2.1.9 with it, so we probably should continue testing Puppet 4 things against that version.